Backends TensorFlow prototyping

nietras opened the pull request:
Create PR to more easily follow changes.

I made the mistake of merging with master, so perhaps "backends" should merge with master too?

nietras commented:
@mdabros added python inspiration solution and the two tensorflow examples we try to replicate.

mdabros commented:
@nietras Yes, in that case, I think we should merge the master branch to the backends branch. There has been an additional update to the master since you merged it with this branch. So we should probably merge the master to the backends branch and then, afterwards, merge the backends branch to this branch. 

I will merge the master to the backends branch. Then you can update this branch afterwards.

mdabros commented:
Good idea with the python solution. We can add the corresponding cntk examples to that solution also.

mdabros commented:
@nietras Can add the python solution to the backends branch in a separate pull request? That way I can branch from it and start adding the cntk python examples.

nietras commented:
Let's just merge the tf branch in, after I merge backends in.

Waiting for windows update to finish 😉

On Jan 25, 2018 09:40, "Mads Dabros" <notifications@github.com> wrote:

> @nietras <https://github.com/nietras> Can add the python solution to the
> backends branch in a separate pull request? That way I can branch from it
> and start adding the cntk python examples.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mdabros/SharpLearning/pull/48#issuecomment-360398402>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKTG7_-gY_KTekdQNRzD0p33_PYBLsUqks5tOD4PgaJpZM4RpePr>
> .
>

nietras commented:
@mdabros merged, I think I am starting to understand what is going on. There is a difference from python to C# that I still do not know why is there, but it is close. Additionally, I have removed using the runner abstraction as that just made things harder to see what is actually going on.

Added also test accuracy evaluation.

All our other backend examples should be modified to match this mnist simple example as close as possible.
